### PR TITLE
Edited Code, the typoes

### PR DESCRIPTION
**Pull request by harderthan**
For understanding clearly, I edited some remarks.
The expert can understand those without any remarks.
But, someone maybe be going to be confused.

Please check this.
Thanks.

### CODE REVIEW

File: test/main.cpp [hunk 1]
```diff
@@ -21,8 +21,8 @@
 TEST(vision_msgs, CreateAABB2D)
 {
   vision_msgs::BoundingBox2D bbox = vision_msgs::createAABB2D(1,2,3,4);
-  EXPECT_FLOAT_EQ(bbox.center.x, 2.5);  // 1 + 3/2
-  EXPECT_FLOAT_EQ(bbox.center.y, 4);    // 2 + 4/2
+  EXPECT_FLOAT_EQ(bbox.center.x, 2.5);  // (1 + 3)/2
```
[the remark this answers is not in the document]
Comment (harderthan) [reply test/main.cpp]: I was stupid...

**Comment by harderthan** [comment]
I re-sent a pull-request.
I edited typoes and parameters for the test functions.

**Comment by Kukanani** [comment]
The original version of the tests pass. However in this case you have changed the expected results of the tests, but the underlying code remains the same, so the new tests will fail. If you want to add clarifying parentheses, I'll happily merge that, but as-is this PR will break the build.

The actual calculation is not `(left + width)/2`, it's `left + (width/2)`, which is equivalent to `left + width/2` using the standard order of operations. See the [implementation](https://github.com/Kukanani/vision_msgs/blob/kinetic-devel/include/vision_msgs/create_aabb.h#L35).

**Comment by mintar** [comment]
I agree that everything was correct before, so no part of this PR should be merged.

**Comment by harderthan** [comment]
Um.. now, I understood.
I thought those are a couple of X, Y points of the bounding box.
Then, these remarks don't need parentheses like `left + (width/2)`. 
Sorry about misunderstanding.
Thanks.

**Pull request closed without merging**